patterns/2-structured/review-committee.md

```diff
@@ -4,7 +4,7 @@ Review Committee
 
 ## Patlet
 
-_TBD_
+The InnerSource working model is a radical departure from more traditional approaches, for developers and managers alike. By establishing a review committee with senior managers of all business units participating in the InnerSource initiative, these managers are more likely to familiarize themselves with InnerSource and provide support for it.
```

Comment:
I think we're missing an important aspect in the patlet, namely that the Review Committee "puts managers mind at ease" because it's a familiar "format" and because it provides them with a certain level of control, without fostering micromanagement. How about the following patlet (keeping the first sentence): 

"By establishing a review committee as an interface between the InnerSource initiative and all senior managers of business units participating in it, the latter are more likely to familiarise themselves with and support the initiative because it affords them a certain level of oversight and control without fostering micromanagement."


It's a bit lengthy, but now it captures the essence of the pattern better, IMHO.

Reply:
I merged your proposal in with only minor tweaks.

Feel free to merge the PR if it looks good now.